silverback/application.py

```diff
@@ -15,9 +15,25 @@
 
 
 class SilverbackApp(ManagerAccessMixin):
+    """
+    The application singleton. Must be initialized prior to use.
+
+    Usages example::
+
+        from silverback import SilverbackApp
+
+        app = SilverbackApp()
+
+        ...  # Connection has been initialized, can call broker methods e.g. `app.on_(...)`
+    """
+
     def __init__(self, settings: Optional[Settings] = None):
         """
         Create app
+
+        Args:
+            settings (Optional[silverback.settings:Settings]): Settings override.
```

Comment:
This link doesn't look right

```suggestion
            settings (Optional[~:class:`silverback.settings.Settings`]): Settings override.
```